source/javascripts/app/_search.js

```diff
@@ -29,22 +29,36 @@
     });
   }
 
+  var wait = (
+      function(){
+        var timeout_handle = 0;
```

Comment:
Is there a reason we need an extra function wrapping all this? Or would something like

```js
var timeout_handle = 0;
var wait = function() {
   // code here
}
```

be equivalent?